tests/io/vasp/test_inputs.py

```diff
@@ -810,10 +810,10 @@ class TestKpoints:
     def test_init(self):
         filepath = f"{VASP_IN_DIR}/KPOINTS_auto"
         kpoints = Kpoints.from_file(filepath)
-        assert kpoints.kpts == [[10]], "Wrong kpoint lattice read"
+        assert kpoints.kpts == [(10,)], "Wrong kpoint lattice read"
         filepath = f"{VASP_IN_DIR}/KPOINTS_cartesian"
         kpoints = Kpoints.from_file(filepath)
-        assert kpoints.kpts == [[0.25, 0, 0], [0, 0.25, 0], [0, 0, 0.25]], "Wrong kpoint lattice read"
+        assert kpoints.kpts == [(0.25, 0, 0), (0, 0.25, 0), (0, 0, 0.25)], "Wrong kpoint lattice read"
```

Comment:
Currently the unit test used `float` for testing. 

Seems some inconsistency needs to be cleaned up, as the `Kpoint` under "line-mode" should actually be a coordinate.